api/v1/routes/iterations.rb

```diff
@@ -14,7 +14,7 @@ class Iterations < Core
           halt 401, { error: message }.to_json
         end
 
-        if !Xapi.exists?(language, slug)
+        if !X::Exercise.exists?(language, slug)
```

Comment:
I like how you used unless below, what do you think about using it here as well?